Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs

```diff
@@ -54,8 +54,6 @@ public CoverageInfo()
             Machines = new HashSet<string>();
             MachinesToStates = new Dictionary<string, HashSet<string>>();
             RegisteredEvents = new Dictionary<string, HashSet<string>>();
-            EventInfo = new EventCoverage();
-            CoverageGraph = new Graph();
```

Comment:
why this change?

Reply:
This change was introduced because of merge. Removing this aligns with the upstream https://github.com/p-org/P/blob/master/Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs#L56